extensions/ArchLinux/modules/skins/timeless.less

```diff
@@ -0,0 +1,96 @@
+@import '../arch_definitions';
+
+/* disable default mediawiki logo  */
+#p-logo {
+    display: none !important;
+}
+
+/* make room for archnavbar and disable fixed searchbar */
+body, #mw-content-container {
+    margin-top: 0;
+    background-color: @body-background-color;
+}
+#mw-content-container {
+    border-bottom: none;
+    padding-top: .5em;
+}
+#mw-header-hack, #mw-header-nav-hack {
+    display: none !important;
+}
+#mw-header-container {
+    position: initial;
+}
+#mw-content {
+    border: @content-border-style;
+}
+#content {
+    border: none;
+    background: transparent;
+}
+
+/* z-order fix */
+#personal-inner {
+    z-index: 200;
+}
+
+/* integrate footer into page background */
+#mw-footer {
+    padding-left: 16em;
+    padding-right: 18em;
+}
+#mw-footer > * {
+    font-size: .8em;
+}
+.mw-footer-container {
+    border-top: none;
+    line-height: 1;
+    color: inherit;
+    box-shadow: none;
+}
+.mw-footer-container a {
+    color:#3366cc;
+}
+.mw-footer-container a:hover {
+    color:#2952a3;
+}
+.mw-footer-container a:visited {
+    color:#2a4b8d;
+}
+.mw-footer-container a:visited:hover {
+    color:#1e3666;
+}
```

Comment:
That is an artifact from testing on mediawiki.org, where these are the generic link colors and I just wasn't able to find it in the less files. Now I have, that should be ```@link. darken( @link, 10% ), @link-visited, darken( @link-visited, 10% )```. How would I go about including ```/skins/Timeless/resources/themes/wikimedia.less```?

Reply:
Are you sure you shouldn't just use `@link-color-*` from [`arch_definitions.less`](https://github.com/archlinux/archwiki/blob/master/extensions/ArchLinux/modules/arch_definitions.less)?

Reply:
I##l use those, I forgot those existed. I'm not sure if I have to go through and restyle all other links as well, though. These are only for the footer because I changed the background color.